apps/aecore/test/aecore_suite_utils.erl

```diff
@@ -1203,3 +1192,37 @@ get_key_hash_by_delta(Node, Delta) ->
     {ok, Header} = rpc(Node, aec_chain, get_key_header_by_height, [TopHeight - Delta]),
     {ok, Hash} = aec_headers:hash_header(Header),
     Hash.
+
+mine_safe_setup(Node, MiningRate, Opts, LoopFun) ->
+    ok = rpc:call(Node, application, set_env,
+                  [aecore, expected_mine_rate, MiningRate], 5000),
```

Comment:
Any particular reason why each timeout is individually hard-coded to `5000`?

Reply:
No, I just didn't mess with the previous setup in that regard. So there might be some reasoning which I don't know about.